### ISSUE TITLE
Sphere overlapping itself?

### ISSUE THREAD

**Issue by RectangleWorld** [processing/processing#3193]
I think the sphere() command may be producing a mesh which may be overlapping itself (but I'm not sure if that's what's causing the artifact I'm seeing). If you use a semi-transparent stroke, you'll notice a bright band compared to the rest of the sphere, probably because of lines drawn on top of each other. I suspect that the loop which draws the sphere goes one incremental step too far in the "theta" angle.

Image - note the bright lines along one band:

![sphere_too_far](https://cloud.githubusercontent.com/assets/1840810/7146101/1e7eca38-e2bf-11e4-9f0d-050ef6f46eab.jpg)

Code:

```Java
import peasy.*;
PeasyCam cam;

void setup() {
  size(800,600,P3D);
  cam = new PeasyCam(this,0,0,0,500);
  background(255);
}

void draw() {
  background(0xFFFFFFFF);
  drawSphere();
}

void keyPressed() {
  saveFrame("sphere_too_far.jpg");
}

void drawSphere() {
  sphereDetail(100,100);
  stroke(0x40FFFFFF);
  fill(0xff000000);
  sphere(400);
}
```

**Comment by JakubValtar** [comment]
Should be fixed in #3211. Please ignore previous pull request, I got lost in the new GitHub interface.

**Comment by benfry** [comment]
Incorporated for 3.0a6.
